src/ensembl/src/content/app/entity-viewer/gene-view/components/gene-filter/GeneFilter.tsx

```diff
@@ -0,0 +1,175 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import { connect } from 'react-redux';
+import { RootState } from 'src/store';
+import { isEntityViewerSidebarOpen } from '../../../state/sidebar/entityViewerSidebarSelectors';
```

Comment:
when a relative import has too many "back up one directory" instructions (and two directories up already feels too many), we start from the src directory:

```
import blah from `src/content/app/entity-viewer/state/sidebar/entityViewerSidebarSelectors`
```